test/python/WMCore_t/MicroService_t/MicroService_t.py

```diff
@@ -24,14 +24,20 @@ def __init__(self, config=None):
         self.config = config
         self.appname = 'test'  # keep it since it is used by XMLFormat(self.app.appname))
 
-    def status(self, serviceName=None):
+    def status(self, serviceName=None, **kwargs):
         "Return current status about our service"
         print("### CALL status API with service name %s" % serviceName)
-        return {'status': "OK", "api": "status"}
+        data = {'status': "OK", "api": "status"}
+        if kwargs:
+            data.update(kwargs)
+        return data
 
-    def info(self, reqName):
+    def info(self, reqName, **kwargs):
         "Return current status about our service"
         print("### CALL info API with request name %s" % reqName)
+        data = {'status': "OK", "api": "info"}
+        if kwargs:
+            data.update(kwargs)
         return {'status': "OK", "api": "info"}
```

Comment:
Shouldn't you `return data` from here too?

Reply:
haa, good catch!